### PR TITLE
Add the SILKY_JSON_ENSURE_ASCII configuration item to support Chinese

### PR DESCRIPTION
**Pull request by eshxcmhk**
Support for users to configure the ```SILKY_JSON_ENSURE_ASCII``` parameter to implement ```json.dumps``` support Chinese

### CODE REVIEW

**Comment by eshxcmhk** [comment]
> Could you add a new tests which sets this settings to false?

I'm sorry, but as a python novice, I don't really know how to add this feature. Also, I don't have time to do this right now, but it works fine on my project

**Comment by nasirhjafri** [comment]
Feel free to do it when you have time. Use this decorator to override settings for a test. https://docs.djangoproject.com/en/dev/topics/testing/tools/#overriding-settings

**Review by auvipy — APPROVED** [review]

File: silk/config.py [hunk 1]
```diff
@@ -28,7 +28,8 @@ class SilkyConfig(metaclass=Singleton):
         'SILKY_PYTHON_PROFILER': False,
         'SILKY_PYTHON_PROFILER_FUNC': None,
         'SILKY_STORAGE_CLASS': 'silk.storage.ProfilerResultStorage',
-        'SILKY_MIDDLEWARE_CLASS': 'silk.middleware.SilkyMiddleware'
+        'SILKY_MIDDLEWARE_CLASS': 'silk.middleware.SilkyMiddleware',
+        'SILKY_JSON_ENSURE_ASCII': True,
```
Comment (auvipy) [inline silk/config.py:32]: Chinese don't work as Unicode?

Comment (eshxcmhk) [reply silk/config.py:32]: > Chinese don't work as Unicode?

The Chinese characters are shown below on the page: 
```
"\u8be5\u8f66\u5728\u3010\u53e4\u853a\u5b8f\u6c38\u3011\u68c0\u6d4b\u672a\u5b8c\u6210\uff0c\u65e0\u6cd5\u767b\u5f55"
```

**Review by auvipy — APPROVED** [review]

File: silk/config.py (hunk 1 above)
Comment (auvipy) [reply silk/config.py:32]: did you encoded your chinease as utf-8?

Comment (eshxcmhk) [reply silk/config.py:32]: > did you encoded your chinease as utf-8?

Yes, the http request send with utf-8 encode

**Pull request merged**